src/BTDeviceModel.h

```diff
@@ -60,14 +67,23 @@ class BTDeviceModel : public QAbstractListModel
      */
     void removeDevice(Device* device);
 
+    void updateItem(const QString& deviceID);
+
     int count();
     Q_SIGNAL void countChanged();
 
     const BTDeviceModel::Device* getDevice(const QString& deviceID) const;
     Q_INVOKABLE QString getDeviceID(int deviceIndex) const;
+
+private:
```

Comment:
This goes against the PImpl pattern, please put all private fields and functions into the Private class in BTDeviceModel.cpp rather than here.